src/util/earthEngine.js

```diff
@@ -90,19 +89,12 @@ const getWorkerInstance = async () => {
     return workerPromise
 }
 
-export const getPeriods = async (eeId) => {
-    const { periodType } = getEarthEngineLayer(eeId)
-
+export const getPeriods = async (eeId, periodType) => {
     const getPeriod = ({ id, properties }) => {
         const year = new Date(properties['system:time_start']).getFullYear()
         const name =
             periodType === 'Yearly' ? String(year) : getStartEndDate(properties)
 
-        // Remove when old population should not be supported
-        if (eeId === 'WorldPop/POP') {
```

Comment:
This eeId is still present in `constants/earthEngine.js`

Reply:
Reverting this removal as we have a saved map from 2016 in the demo db that still uses this dataset.